auxlib/packaging.py

```diff
@@ -1,101 +1,141 @@
-# #!/usr/bin/env python
-# # -*- coding: utf-8 -*-
-# import logging
-# import os
-# import re
-# import subprocess
-# import pkg_resources
+# -*- coding: utf-8 -*-
+from __future__ import print_function, division, absolute_import
+from logging import getLogger
+from os.path import basename, dirname, join, isdir
+from re import match
+from setuptools.command.build_py import build_py
+from setuptools.command.sdist import sdist
+from setuptools.command.test import test as TestCommand
+from subprocess import CalledProcessError, check_call, check_output
+from sys import exit
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Redefining built-in 'exit'](https://www.codacy.com/app/kalefranz/auxlib/file/2131695316/issues/source?bid=3138661&fileBranchId=3139130#l10)